app/test_main.py

```diff
@@ -2,2 +1,47 @@
+import pytest
 
-# write your code here
+from .main import get_human_age
+
+
+def test_add_not_int_argument() -> None:
+    with pytest.raises(TypeError):
+        get_human_age(12.52, "52.12")
+
+
+def test_negative_value() -> None:
+    assert (get_human_age(-12, -2) == [0, 0]), \
+        "The age shouldn't be negative"
+
+
+def test_get_human_age_should_return_list_two_elem() -> None:
+    assert isinstance(get_human_age(16, 15), list)
+
+
+@pytest.mark.parametrize(
+    "cat_to_human_years,dog_to_human_years,result",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (24, 24, [2, 2]),
+        (28, 28, [3, 2]),
+        (29, 29, [3, 3]),
+        (100, 100, [21, 17]),
```

Comment:
The parameterized test `test_functionality` covers a range of scenarios, which is excellent. However, consider adding cases for very large numbers to ensure the function handles them correctly, as suggested in the checklist.